app/controllers/availability_indicators_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
     documents = search_service.fetch(document_ids, rows: document_ids.count).first.documents
     documents.each do |document|
       availability = availability_data.present? ? availability_data[document.id] : nil
-      indicator = render_to_string('catalog/_availability_indicator', layout: false, locals: { xray: false, document: document, doc_avail_values: availability })
```

Comment:
I realized that not only does adding `xray:false` not work, the gem itself is only available in development, which does not affect any of these features in production.